config/application.rb

```diff
@@ -30,6 +30,9 @@ class Application < Rails::Application
 
     config.api_only = true
 
+    # This prevents rails from escaping html like & in links when working with JSON
+    config.active_support.escape_html_entities_in_json = false
```

Comment:
@ayaleloehr I know you had some reluctance to let this ride because it poses potential XSS risks, but at least based on how it is used here that risk is 100% limited to doing XSS on oneself as the only way to do it would be to pass in query parameters that create links that when clicked launches a script. We're not persisting anything and presenting it to other users.

I'd like to think brakeman would catch any dangerous places where this could become a problem.